crates/uv-resolver/src/resolver/batch_prefetch.rs

```diff
@@ -52,7 +52,7 @@ impl BatchPrefetcher {
         index: &InMemoryIndex,
         selector: &CandidateSelector,
     ) -> anyhow::Result<(), ResolveError> {
-        let PubGrubPackage::Package(package_name, _, _) = &next else {
+        let PubGrubPackage::Package(package_name, None, None) = &next else {
```

Comment:
Why `None` for the extras field?

Reply:
Oh sorry

Reply:
If we include an extra, we'll also include the non-extra variant in the tree. So it seems fine to confine this to those non-extra variants, rather than duplicating the prefetch.

Reply:
okee